### DISCUSSION TITLE
Standup (Wednesday, May 18, 2022)

### DISCUSSION THREAD

**Discussion by CristianCazares** [CarlosCRG19/ijalti-peers#66] [General]
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Basic chip implementation
 - Meeting with front-end team

**What are your priorities for the next 24 hours?**:
 - Finish chip handling

**Blockers**:
 - Components re-render indefinitely

**Shoutouts**:
 - @antonieto for moving out I'm sorry I didn't help you :'(

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Revisit the client progress
 - Meeting with front team

**What are your priorities for the next 24 hours?**:
 - Custom sytles
 - City select

**Shoutouts**:
 - @DavBot02  @paotg123

**Comment by paotg123** [comment]
**What did you achieve in the last 24 hours?**:
 - I meet again with the front team

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -I can´t open the file

**Shoutouts**:
@DavBot02

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Dicussed next steps with @antonieto 
 - Start working on my on Completing company and job offer Models

**What are your priorities for the next 24 hours?**:
 -  Finished those models

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto  for moving to his new appartment
